src/extractor/compressed_edge_container.cpp

```diff
@@ -134,7 +135,21 @@ void CompressedEdgeContainer::CompressEdge(const EdgeID edge_id_1,
     // weight1 is the distance to the (currently) last coordinate in the bucket
     if (edge_bucket_list1.empty())
     {
-        edge_bucket_list1.emplace_back(OnewayCompressedEdge{via_node_id, weight1, duration1});
+        if (weight1 >= INVALID_SEGMENT_WEIGHT)
```

Comment:
better `, clip(weight1), ` with `clip = [&clipped_weights] (EdgeWeight) -> SegmentWeight ` closure to reuse in both places